src/dra/5298C.c

```diff
@@ -906,41 +906,40 @@ void DrawTimeAttackMenu(MenuContext* ctx) {
     }
 }
 
-#ifndef NON_MATCHING
-INCLUDE_ASM("dra/nonmatchings/5298C", DrawSettingsButton);
-#else
-extern u8 c_chPlaystationButtons[];
-extern u8 c_chShoulderButtons[];
-
 void DrawSettingsButton(MenuContext* ctx) {
     s32 cursorX;
     s32 i;
     s32 x;
-    s32 y;
     s32 buttonId;
     s32 btn1_x;
     s32 btn2_x;
-
+#if defined(VERSION_HD)
+    cursorX = 0x80;
+#define W 0x28
+#define XVAR cursorX
+#elif defined(VERSION_US)
     cursorX = 0x98;
```

Comment:
My opinion is that having a macro for the value is a little cleaner for cursorX. `cursorX = CURSORX_VAL;` https://decomp.me/scratch/G4Qv4

Reply:
Sounds reasonable to me, change incoming

Reply:
I don't actually know the difference between a define and a const, if const is preferred I'll change it